web/legacy/api/images/verify_app.ts

```diff
@@ -126,21 +126,6 @@ export const handleVerifyApp = async (
     (metadata) => metadata.verification_status === "verified",
   );
 
-  // Check if app is allowed to be app store and world app approved
```

Comment:
@andy-t-wang I think we should keep it, or am I missing something?

Reply:
Yeah lets get rid of this of the changes in this file, I made these changes in dev-portal-2 not in main

Reply:
I've revert this